### DISCUSSION TITLE
Do you support plate.js

### DISCUSSION THREAD

**Discussion by DineshCodeFlow** [liveblocks/liveblocks#1983] [Help]
Hello I am working a plate.js editor and we are not looking for a solution for realtime update. 
I was trying to search an example using  liveblocks in plate.js.

Do you have any?

**Comment by jrowny** [comment]
Hi there,

Platejs has hardcoded their collaboration plugin to only work with one specific provider. You would need to fork the Yjs plugin for PlateJS so you can pass the Liveblocks provider. Internally Plate uses Slate, and Slate's YJS plugin is agnostic towards the provider. This guide may be helpful: https://liveblocks.io/docs/get-started/yjs-slate-react
